src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -411,6 +411,9 @@ def makeTransferRequest(self, wflow, rseList):
             if not isRelVal(wflow.data):
                 # enforce RSE quota
                 rses = list(set(rseList) & self.rseQuotas.getAvailableRSEs())
+            else:
+                rses = self._getValidSites(wflow, dataIn)
```

Comment:
I think this should be modified to:
```
rses = rseList
```

assuming that `rseList` is properly constructed and passed to this function for RelVal and for Non-RelVal workflows.

Reply:
ok